### ISSUE TITLE
Focus on add task field when opening popup

### ISSUE THREAD

**Issue by itsonlycode** [sgaraud/gnome-extension-taskwarrior#12]
I suggest that the add task field has focus when opening menu (I use <ctrl><f8>) so it is possible to add tasks without moving to the mouse.

**Comment by sgaraud** [comment]
good point indeed. In fact I would like to have also the possibility to mark tasks as done without mouse click, only with keyboard.
I will look into that asap.
